NMDSdevopsServiceAdm/DataEngineering
# Add minimum value check to validation

# Description
Add the utils functions and tests to create a check of min values.
Add the rules for the IND CQC merged dataset min values

# How to test
Unit tests passing
Run in branch here: https://eu-west-2.console.aws.amazon.com/states/home?region=eu-west-2#/v2/executions/details/arn:aws:states:eu-west-2:344210435447:execution:validation-add-has-min-check-Ind-CQC-Filled-Post-Estimates-Pipeline:aa063c31-7a1b-4a07-a728-f11444ce381f

# Developer checklist
